setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = -vs --reuse-db --showlocals --tb=short
+addopts = -vs --reuse-db --showlocals --tb=short --ignore=src/olympia/amo/tests/ui
```

Comment:
This is just to make Travis not run these tests, and may be temporary if we decide not to use CircleCI.